.github/scripts/instr-count-benchmark/create-nightly-env.sh

```diff
@@ -10,6 +10,11 @@ set +a;
 conda create -y -q --name ${CONDA_ENV_NAME} python=${PYTHON_VERSION}
 . activate ${CONDA_ENV_NAME}
 
+# We need Valgrind to collect instructions.
+conda install -y valgrind -c conda-forge
```

Comment:
It's nice to know valgrind is in the conda-forge repo!

Reply:
Indeed!